docs/code_of_conduct.md

```diff
@@ -2,11 +2,11 @@
 
```

Comment:
@sanchibansal340 could you make sure the Code of Conduct's content is consistent with this one https://github.com/anitab-org/documentation/blob/master/Contributing/CODE_OF_CONDUCT.md?

Reply:
@isabelcosta I will make the changes needed